docs/source/faq.md

```diff
@@ -0,0 +1,19 @@
+# Frequently Asked Questions (FAQ)
+
+A collection of frequently asked questions with answers!
+
+If you have a question & have found an answer, send a PR to add it here!
+
+## Can I use repo2docker to bootstrap my own Dockerfile?
+
+If you pass the `--debug` flag to `repo2docker`, it outputs the intermediate
```

Comment:
Lead with the answer: "No you can't."

Reply:
Should we be consistent and use `jupyter-repo2docker` when referring to the command line program?

Reply:
@betatim https://github.com/jupyter/repo2docker/pull/121 is the fix for the jupyter-repo2docker / repo2docker inconsistentcy I think.